js/components/frame.js

```diff
@@ -688,7 +688,10 @@ class Frame extends ImmutableComponent {
       contextMenus.onShowAutofillMenu(e.suggestions, e.rect, this.frame)
     })
     this.webview.addEventListener('hide-autofill-popup', (e) => {
-      windowActions.autofillPopupHidden(this.props.tabId)
+      let webContents = this.webview.getWebContents()
```

Comment:
this shouldn't be necessary anymore. I changed windowStore to make a check instead. I think we may want to revert this unless I'm missing something